### DISCUSSION TITLE
Keep note tree scroll position after opening note in new tab

### DISCUSSION THREAD

**Discussion by dymani** [TriliumNext/Trilium#4031] [Ideas] [answered]
Hi,

I need to reference multiple notes for work and my usual routine is this: I scroll through the note tree (which is quite long), then open the notes I need in new tabs with middle clicks.

However, after opening each note in a new tab, the note tree always automatically scroll to the current active note, which may be quite far away in the tree. I find this quite inconvenient since I need to constantly scroll back and forth in the note tree.

**Example**
Let's say I am at `Inbox / Book to read` and I want to open `Journal / 2021 / 12 - December / 21 - Thursday / Christmas shopping` and `Journal / 2021 / 12 - December / 21 - Thursday / Office party` in new tabs.

![2023-06-17_02-27-07](https://github.com/zadam/trilium/assets/10493889/269ecefa-5d0b-4080-9a1a-7d1ad9e004ba)

After opening the first note, I need to scroll again to find the second note.

Would it be more user-friendly if the note tree stays at the same position when a note is opened in a new tab?

Thanks!

**Comment by Nriver** [comment]
The middle key method also does not work. emm

**Reply by Nriver** [reply]
It's like the "Always select open file" feature in Jetbrains IDE's file tree. It would be nice if it's also configurable in Trilium.

**Comment by zadam** [comment, marked as answer]
Created an issue for this and fixed : https://github.com/zadam/trilium/issues/4041
